examples/calendar-widget/package.json

```diff
@@ -3,6 +3,6 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "build": "rm -rf ./dist/ && rsync -r . ./dist/"
+    "build": "rm -rf ./dist/ && mkdir dist && cp -r `ls -A | grep -v 'dist'` ./dist/"
```

Comment:
what was wrong with rsync?

Reply:
`rsync` is not available on CircleCI

Reply:
are all examples built on circle? don't we just want the one for the test?